app/controllers/steps/conviction/compensation_paid_amount_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Steps
+  module Conviction
+    class CompensationPaidAmountController < Steps::ConvictionStepController
+      def edit
+        @form_object = CompensationPaidAmountForm.new(
```

Comment:
If you want you can probably replace these lines with a call to `build(current_disclosure_check)` like in the other controller (CompensationPaymentReceiptController).
But they behave the same, they are equivalent in this case.